zk/src/archive/web/js/zk/widget.js

```diff
@@ -2161,6 +2162,10 @@ wgt.$f().main.setTitle("foo");
 	 * @return zk.Widget this widget.
 	 */
 	setZIndex: _zkf = function (zIndex, opts) {
+		if (!opts.floatZIndex)
+			this.userZIndex = true;
+		if (!zIndex)
+			delete this.userZIndex;

```

Comment:
1. Use `this._userZIndex`
2. `this._userZIndex = false`

Reply:
ok